### PR TITLE
Setupdb

### PR DESCRIPTION
**Pull request by rabe200**
compare to:
[userStory1](https://github.com/rabe200/Capstone_Tarot_App/issues/2)

artworks not uploaded yet, so don't expect pictures

### CODE REVIEW

File: public/images/2c.jpg
Comment (toshydev) [inline public/images/2c.jpg:1]: 😍

**Review by toshydev — CHANGES_REQUESTED** [review]
Two suggestions, see comments 📝

File: pages/_app.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
 import GlobalStyle from "../styles";
+import { SWRConfig } from "swr";
+
+const fetcher = (url) => fetch(url).then((r) => r.json());
```
[the remark this answers is not in the document]
Comment (rabe200) [reply pages/_app.js]: @toshydev thanks a lot, i picked up both suggestions and also used swrConfig the proper way now by using its globalstate function :) wouldn't have seen that faux pas without your hints!

**Comment by manukao** [comment]
very nice, just one thing: in your user story you have "On refresh a random card is displayed" as an acceptance criteria - on the initial page load a random card is shown, on click as well, but on refresh this is not happening - so if you want a random card shown after each page refresh you should update your code.

**Comment by rabe200** [comment]
@manukao that is true. nice that you have seen it. very attentive. but i thought it would be a better user experience if it doesn't reload on refresh, but by clicking on the picture (placeholder atm). maybe i should change the user story in that terms. :) thanks for your review!

**Pull request merged**